monitor-function-calls/SETUP.md

```diff
@@ -1,55 +1,68 @@
-
-6. `contracts` (required) - The Object value for this key corresponds to contracts that we want to monitor function
-calls for. Each key in the Object is a contract name that we can specify, where that name is simply
-a string that we use as a label when referring to the contract (the string can be any valid string
-that we choose, it will not affect the monitoring by the agent). The Object corresponding to each
-contract name requires an address key/value pair, abi file key/value pair, and a `functions` Object
-containing the names of functions as keys and Objects containing Finding types and severities as values. For
-example, to monitor if `createPool` was called in a Uniswap V3 Factory contract, we would need the
-contract address, the ABI saved locally as a JSON formatted file, the exact function name corresponding
-to what is listed in the ABI file, and a type and severity for the alert:
-
-```
+2. `developerAbbreviation` (required) - Type in your desired abbreviation to specify your name or
+your development team name.  For example, Arbitrary Execution uses the abbreviation `"AE"` for its
+`developerAbbreviation` value.
+
+3. `protocolName` (required) - Type in the name of the protocol.  For example, for the Uniswap
+protocol you may type in `"Uniswap"` or `"Uniswap V3"`, for the Sushi Swap protocol you may type in
+`"Sushi"` or `"SushiSwap"`, etc.
+
+4. `protocolAbbreviation` (required) - Type in an appropriate abbreviation for the value in
+`protocolName`.  For example, `"Uniswap"` may be abbreviated `"UNI"` and `"Sushi Swap"` may be
+abbreviated `"SUSH"`, etc.
+
+5. `contracts` (required) - The Object value for this key corresponds to contracts that we want to
+monitor function calls for. Each key in the Object is a contract name that we can specify, where
+that name is simply a string that we use as a label when referring to the contract (the string can
+be any valid string that we choose, it will not affect the monitoring by the agent). The Object
+corresponding to each contract name requires an address key/value pair, abi file key/value pair, and
+a `functions` key.  The corresponding value for the `functions` key is an Object containing the
+names of functions as keys. The value for each function name is an Object containing:
+    * type (required) - Forta Finding Type
+    * severity (required) - Forta Finding Severity
+    * expression (optional) - A string that can be evaluated as a condition check when a function is
+    called.  The format of the expression is `<argument_name> <operator> <value>` (delimited by
+    spaces) where `argument_name` is the case-sensitive name of an input argument, specified in the
+    ABI, that is provided as part trace data when the function is called, `operator` is a standard
+    operation such as: `>=, !==, <` (a full table on supported operators can be found in the
+    [Expression Compatibility Table](#expression-compatibility-table)), and `value` is an address,
```

Comment:
The link is broken (since the header and table itself doesn't exist), you can either add in the table or remove this link